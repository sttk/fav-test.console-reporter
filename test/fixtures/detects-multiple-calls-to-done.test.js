'use strict';

if (typeof setImmediate !== 'function') { // Node.js v0.8
  global.setImmediate = function(fn) {
    setTimeout(fn, 0);
  };
}

it('double done', function(done) {
  // Calling `done()` twice is an error
  setImmediate(done);
  setImmediate(done);
});

