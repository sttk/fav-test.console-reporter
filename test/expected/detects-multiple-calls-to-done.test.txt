

  âœ“ double done
  1) double done


  1 passing (4ms)
  1 failing

  1) double done:
     Error: done() called multiple times
      at processImmediate (internal/timers.js:461:21)



